src/@next/CurrencyInput/CurrencyInput.tsx

```diff
@@ -9,13 +8,14 @@ export type CurrencyInputProps = Omit<
   locale?: string;
   value?: number;
   onChange?: (value: number) => void;
+  prefix?: string;
```

Comment:
The prefix will be filled with currencySymbol not currencyCode, ex: $ not USD

Reply:
Based on this Stackoverflow QnA https://stackoverflow.com/questions/50650503/get-the-currency-symbol-for-a-locale, it needs both locale and currencyCode to get the currency symbol

Reply:
Oh yeah from this prop I was under the assumption the prefix will be whatever which passed (user will need to pass "$") to display dollar sign. Ah okay, I think I should mentioned passing currencySymbol instead of currencyCode for the overriding part